client/src/actions/hanabiActions.js

```diff
@@ -0,0 +1,30 @@
+export function resetGame(){
+  return {type: "RESET_GAME"}
+}
+
+export function addPlayer(playerName){  
+  return {type: "ADD_PLAYER", playerName}
+}
+
+export function startGame(originalPlayers, originalDeck){
+  const deck = [...originalDeck];
+  let cardCount = undefined;
+  if (originalPlayers.length > 1 && originalPlayers.length < 4){
```

Comment:
I'm going to opt to _not_ do this just yet as I don't see this logic repeating.

We need to change range for number of players, but beyond that we'll be running checks for 'next number' and filtering for same number or same color. 

If this _is_ needed, lets abstract later.